studio/src/editors/merch-card-editor.js

```diff
@@ -167,6 +171,28 @@ class MerchCardEditor extends LitElement {
         );
     }
 
+    updated() {
+        this.toggleFields();
+    }
+
+    async toggleFields() {
+        if (!this.fragment) return;
+        const merchCardCustomElement = await merchCardCustomElementPromise;
+        if (!merchCardCustomElement) return;
+        this.querySelectorAll('sp-field-group.toggle').forEach((field) => {
+            field.style.display = 'none';
+        });
+        const variant = merchCardCustomElement.getFragmentMapping(
+            this.fragment.variant,
+        );
+        if (!variant) return;
+        Object.keys(variant).forEach((key) => {
+            const field = this.querySelector(`sp-field-group.toggle#${key}`);
+            if (field) field.style.display = 'block';
+        });
+        this.superWide = variant.size?.includes('super-wide');
```

Comment:
might be trivial but some variants have other sizes like `wide`

Reply:
@Axelcureno wide is supported by default. this flag is to show hide the extra-wide option when supported by the variant.

Reply:
I also added flag for wide cards.